sql_assignment/query.sql

```diff
@@ -0,0 +1,49 @@
+CREATE DATABASE TEST;
+USE TEST;
+
+CREATE TABLE Employee (EmployeeID int primary key, FirstName varchar(10), LastName varchar(10), Dept varchar(10), Salary int );
```

Comment:
Why all string columns length is 10 ?

Reply:
I thought FirstName, LastName, Dept will have characters less than 10